stable/kommander/templates/grafana/dashboards/prometheus-remote-write.yaml

```diff
@@ -0,0 +1,1414 @@
+{{- if .Values.grafana.enabled }}
```

Comment:
I think this might be because remote-write isn't set up in prometheus. if customer configured prom with remote-write, I assume the metrics would show up

Reply:
ah, fair point